client/verta/verta/operations/monitoring/monitored_entity.py

```diff
@@ -63,13 +63,18 @@ def _get_proto_by_id(cls, conn, id):
         return conn.maybe_proto_response(response, Message.Response).monitored_entity
 
     @classmethod
-    def _get_proto_by_name(cls, conn, name, workspace):
-        # NOTE: workspace is currently unsupported until https://vertaai.atlassian.net/browse/VR-9792
-        Message = _DataMonitoringService.GetMonitoredEntityByNameRequest
-        msg = Message(name=name)
-        endpoint = "/api/v1/monitored_entity/getMonitoredEntityByName"
-        response = conn.make_proto_request("GET", endpoint, params=msg)
-        return conn.maybe_proto_response(response, Message.Response).monitored_entity
+    def _get_proto_by_name(cls, conn, name, workspace=None):
+        Message = _DataMonitoringService.FindMonitoredEntityRequest
+        msg = Message(names=[name], workspace_name=workspace, page_number=1, page_limit=-1)
+        endpoint = "/api/v1/monitored_entity/findMonitoredEntity"
+        response = conn.make_proto_request("POST", endpoint, body=msg)
+        results = conn.maybe_proto_response(response, Message.Response)
+        count = results.total_records if results else 0
+        if count > 1:
+            print("Warning: found more than one monitored entity with name {}".format(name))
```

Comment:
I'd lean towards using `warnings` unless there are reasons not to
```suggestion
            warnings.warn(
                "found more than one monitored entity with name {}".format(name)
            )
```